### PR TITLE
add harvest info in factory vaults

### LINKED ISSUE #324: Add harvesting difference between factory and trad vaults
https://hackmd.io/@dr00shie/harvest-docs

### PR DESCRIPTION
**Pull request by MarcoWorms**
closes #324

### CODE REVIEW

File: docs/getting-started/products/yvaults/vault-factory.md [hunk 1]
```diff
@@ -43,21 +43,36 @@ Factory-deployed yVaults for Curve LP tokens contain up to three ready-made yiel
...
+
+Harvests in factory strategies still rebalance debt between strategies and allocate idle funds to strategies, while also pulling accumulated rewards into the strategy contract to be swapped by the ySwaps team. Once swaps are completed, which may take several days, anyone can call harvest a second time to acknowledge the profit generated by the swap and distribute it to the vault making pricePerShare increase.
+
+Although harvests will be permissionless for this specific group of vaults, Yearn will continue utilizing standard keeper automation to call harvests, even if no one else does. These automated harvests are run by keep3r.network and are often dependent on network congestion and transaction viability. For example, if the network conditions are consistently unfavorable, there may be delays between harvests.
+
+### Calling Harvest
+
+To call harvest, use the harvest() function on the keeper found at keeper.factory.ychad.eth. Simply enter the strategy address from the factory vault you wish to call harvest on and click “write.”
```
Comment (0xBasically) [inline docs/getting-started/products/yvaults/vault-factory.md:64]: you deleted the link here [keeper.factory.ychad.eth](https://etherscan.io/address/keeper.factory.ychad.eth#writeContract)

**Review by 0xBasically — CHANGES_REQUESTED** [review]
Other than my comment lgtm

File: docs/getting-started/products/yvaults/vault-factory.md (hunk 1 above)
Comment (MarcoWorms) [reply docs/getting-started/products/yvaults/vault-factory.md:64]:
```suggestion
To call harvest, use the harvest() function on the keeper found at [keeper.factory.ychad.eth](https://etherscan.io/address/keeper.factory.ychad.eth#writeContract). Simply enter the strategy address from the factory vault you wish to call harvest on and click “write.”
```

**Review by 0xBasically — APPROVED** [review]
lgtm

**Pull request merged**